tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -2095,116 +1870,79 @@ def test_deletes_during_kserver_pods_down(self):
-                                                               hostname=hostname)
-            LOGGER.debug("Services status on %s : %s", pod_name, resp)
-            if not resp[0]:
-                counter += 1
-            server_data_pod_list.remove(pod_name)
-        assert_utils.assert_equal(counter, 0, "Services on some server pods not stopped.")
-        LOGGER.info("Step 4: Services of deleted server pods are in offline state")
-
-        LOGGER.info("Step 5: Check services status on remaining pods %s", server_data_pod_list)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
-                                                           fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services of remaining pods are in online state")
-        event.clear()
-
+        self.restore_method = const.RESTORE_DEPLOYMENT_K8S
+        self.restore_pod = True
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown random %s (K) server pod %s one by one during "
+                    "WRITEs in background. Verified cluster and services states are as expected &"
+                    " remaining pods status is online.", self.kvalue, self.pod_name_list)
         LOGGER.info("Joining background thread. Waiting for %s seconds to "
                     "collect the queue logs", HA_CFG["common_params"]["60sec_delay"])
         thread.join()
         responses = dict()
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
-        LOGGER.info("Step 6: Verify status for In-flight WRITEs while %s (K) server pods going "
+        LOGGER.info("Step 3: Verify status for In-flight WRITEs while %s (K) server pods going "
                     "down should be failed/error.", self.kvalue)
         pass_logs = list(x[1] for x in responses["pass_res"])
         LOGGER.debug("Pass logs list: %s", pass_logs)
         fail_logs = list(x[1] for x in responses["fail_res"])
         LOGGER.debug("Failed logs list: %s", fail_logs)
         resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
-        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        assert_utils.assert_false(len(resp[1]),
+                                  f"Expected Pass, But Logs which contain failures: {resp[1]}")
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
-                                 f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 6: Verified status for In-flight WRITEs while %s (K) server pods "
+                                 f"Logs which contain passed IOs: {resp[1]}")
```

Comment:
Here also TODO should be added.
# TODO: Expecting Failures when server pods going down. Re-test once CORTX-28541 is Resolved